pkg/alertmanager/amcfg_test.go

```diff
@@ -1407,7 +1407,7 @@ templates: []
 							Receiver:          "test",
 							MuteTimeIntervals: []string{"test"},
 						},
-						MuteTimeIntervals: []monitoringv1alpha1.MuteTimeInterval{
```

Comment:
It would be nice to extend the tests to test the new options

Reply:
Done